packages/components/asset/src/Asset.styles.ts

```diff
@@ -15,6 +15,7 @@ export function getAssetStyles() {
       height: '100%',
       maxWidth: '100%',
       maxHeight: '100%',
+      objectFit: 'contain',
```

Comment:
I think it looks good, but it would be nice to see a story with an image that would be distorted without this fix. So we have a test for this case and so we can compare prev/now